packages/boxel-motion/addon/behaviors/base.ts

```diff
@@ -41,15 +41,13 @@ export default interface Behavior {
    *
    * @param options
    */
-  toFrames(
+  getFrames(
     options:
       | EasingToFramesArgument
       | SpringToFramesArgument
       | StaticToFramesArgument
-      | WaitToFramesArgument,
-    interpolator?: (from: any, to: any, t: number) => any,
-    serializer?: (value: any) => any
-  ): Frame[];
+      | WaitToFramesArgument
+  ): Generator<Frame | void>;
```

Comment:
```suggestion
): Generator<Frame | void, void, never>;
```

Reply:
Could you double check this if this is needed?

Reply:
I don't think this is strictly necessary, but it probably is good to do to prevent accidental introduction of return values in the generators.